configure.ac

```diff
@@ -43,10 +43,8 @@ AC_CHECK_SENDFILE
 
 AC_ARG_WITH([eprefix], [AS_HELP_STRING([--with-eprefix], [path for Gentoo/Prefix project])])
 # ensure eprefix ends with a slash, since the code base expects that
-case "$with_eprefix" in
-	*/) with_eprefix="$with_eprefix" ;;
-	*)  with_eprefix="${with_eprefix}/" ;;
-esac
+with_eprefix="${with_eprefix%/}/"
```

Comment:
this would require posix shell, but configure scripts should be /bin/sh compatible

Reply:
where do I know what is "/bin/sh compatible" but not POSIX?

Reply:
https://www.gnu.org/software/autoconf/manual/autoconf-2.69/html_node/Shell-Substitutions.html

scroll down to the paragraph for ${var%word}